main.scss

```diff
@@ -1,14 +1,173 @@
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold, $style: italic);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium, $style: italic);
+
+	@include oFontsInclude(FinancierDisplayWeb);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium);
+
+	:root {
+		--font-family-sans-serif: 'MetricWeb';
+		--font-family-serif: 'FinancierDisplayWeb';
+
+
+		// primary text color
+		--palette-text-primary: #{inspect(oColorsGetPaletteColor('black-80'))};
+
+
+		// error message (ex: edit time has expired)
+		// background
+		--palette-error-light: #{inspect(oColorsGetPaletteColor('claret-80'))};
+		// border-left
+		--palette-error-darkest: #{inspect(oColorsGetPaletteColor('claret-70'))};
+
+
+		// community guidelines box
+		// background
+		--palette-primary-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+		// border
+		--palette-primary-light: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// apply button in configure tab
+		--palette-success-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+		--palette-success-light: #{inspect(oColorsGetPaletteColor('teal-60'))};
+
+
+		// stream section selected tab item, selected item count badge
+		--palette-primary-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// stream section unselected tab comment count badge
+		// blockquote background
+		--palette-grey-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// stream section tab bar divider
+		--palette-divider: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// respect, reply, share, report
+		--palette-grey-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// report button "reported" state
+		--palette-error-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// moderation drop down
+		// "Approved" button (approve button state)
+		--palette-success-dark: #{inspect(oColorsGetPaletteColor('black'))};
+		// "Un Feature" button (feature button state)
+		--palette-primary-dark: #{inspect(oColorsGetPaletteColor('black'))};
+	}
+
+	// Mainly for the links in the community guidelines box
+	a {
+		@include oTypographyLink;
+	}
+
+	// remove divider of the top tab bar
+	.coral-tabBar {
+		border-bottom-width: 0;
+	}
+
+	// top tab bar buttons
+	.coral-tabBar-allComments,
+	.coral-tabBar-myProfile,
+	.coral-tabBar-configure	{
+		@include oButtons;
+
+		border-width: 1px;
+
+		// normally, this should come for free with oButtons mixin
+		// but we have to declare this explicitly here because
+		// Coral styles override our styles for some reason
+		&[aria-selected=true],
+		&[aria-pressed=true],
+		&[aria-current] {
+			color: oButtonsGetColor('', 'color', 'primary') !important;
+			background-color: oButtonsGetColor('', 'background', 'primary') !important;
+			border-color: oButtonsGetColor('', 'border', 'primary') !important;
+			border-top: 1px !important;
```

Comment:
I think if we make these borders `border-top-width: 1px` & `border-bottom-width: 1px` we wouldn't need the `!important`

Reply:
We had a chat in real life and decided not to change this.